### ISSUE TITLE
[Adaptive Cards- Import- Food Order]: Talkback does not announces the role of 'Images' present on 'Food Order' pane.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7258]
https://microsoft.visualstudio.com/OS/_workitems/edit/34086249
